projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -193,20 +186,13 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		}
 
 		setShowRetry( false );
-		setIsWaitingForPreviousBlock( false );
 		setErrorMessage( false );
-		setNeedsMoreCharacters( 0 );
+		setNeedsMoreCharacters( false );
 		setIsLoadingCompletion( true );
 
 		const data = {
-			content: createPrompt( currentPostTitle, contentBefore, categoryNames ),
-			post_id: postId,
+			content: createPrompt( currentPostTitle, contentBefore, categoryNames, tagNames ),
 		};
-
-		tracks.recordEvent( 'jetpack_ai_gpt3_completion', {
```

Comment:
Please dont remove the track s event

Reply:
It has to have been an issue when I merged trunk into it. I didn't remove that on purpose...